build.ps1

```diff
@@ -9,7 +9,7 @@ param (
     [string]$SemanticVersion = '1.0.0-zlocal',
     [string]$Branch,
     [string]$CommitSHA,
-    [string]$BuildBranch = '1c8734ee61e209f159972ab974784ba55ee2bd6d'
+    [string]$BuildBranch = '74ca7aac689290660786a9e295da666eb7c13c2e'
```

Comment:
This will be switched to a `master` commit once https://github.com/NuGet/ServerCommon/pull/279 is merged.